Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -816,7 +816,7 @@ void GStreamerMediaEndpoint::doSetLocalDescription(const RTCSessionDescription*
         }
 
 #ifndef GST_DISABLE_GST_DEBUG
-        auto dotFileName = makeString(unsafeSpan(GST_OBJECT_NAME(m_pipeline.get())), ".setLocalDescription"_s);
+        auto dotFileName = makeString(String::fromUTF8(GST_OBJECT_NAME(m_pipeline.get())), ".setLocalDescription"_s);
```

Comment:
Same here, maybe this could be `CStringView::unsafeFromUTF8()`?

Reply:
...and I see below that there are many other cases where this would apply, and all use `makeString()`. Could it be that `makeString()` does not have support for `CStringView`? It would seem like a good thing to me to make sure it can accept `CStringView` instances.